### PR TITLE
Attack weather api using requests

### PR DESCRIPTION
**Pull request by kwiznia**
Attack the open weather api (http://openweathermap.org/current) using requests and lettuce.

### CODE REVIEW

**Comment by jframos** [comment]
Use the project structure documented here: http://lettuce.it/tutorial/simple.html#tutorial-simple
Rename the PR with a better name (related to what you want to do), and write a PR description.

**Comment by jframos** [comment]
I recommend writting a message to describe each commit.

File: PycharmProjects/weather2/test/weather2.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    """
+    def getStatus_code(self, response):
+        world.response = response
+        world.response = requests.get(response)
+        return world.response.status_code
+    """
+
+    def getInformation(self, response):
+        world.response = response
+        world.response = requests.get(response)
```
Comment (jframos) [inline PycharmProjects/weather2/test/weather2.py:15]: Review lines: 15 and 16. What is "wrong"?

File: PycharmProjects/weather2/test/weather2steps.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+@step("I check if the latitude and longitude are correct")
+def check_lat_and_long_are_correct(self):
+    latlon = world.page_info['coord']
+    assert latlon['lon'] != ""
+    assert latlon['lat'] != ""
+    assert_equals(int(world.expectedLatitude), latlon['lat'])
+    assert_equals(int(world.expectedLongitude), latlon['lon'])
+
+
+@step("I check if the status code of the page look by latitude and longitude is 200")
```
Comment (jframos) [inline PycharmProjects/weather2/test/weather2steps.py]: I think you could have only one common method: Lines 61 and 35

File: README.md [hunk 3]
```diff
@@ -1 +0,0 @@
-# ExampleRepo
```
Comment (jframos) [inline README.md:1]: Add a brief outline for the implemented test cases in order to understand what contains this repository.

**Comment by kwiznia** [comment]
This branch has one feature and to .py files. This includes a few test to check if the weather api works properly. There is one feature with four scenarios.  Two of them checks if the page works properly given a country and city on one hand and latitude and longitude on the other. 
The other two check if the temperature is correct given the same parameters as before on each cases.

**Comment by jframos** [comment]
LGTM
We will improve it in next releases :)

**Pull request merged**